src/App.css

```diff
@@ -1,35 +1,198 @@
-.Nav-bar {
-  display: flex;
-  justify-content: flex-start;
-  align-items: center;
-  gap: 2vw;
-  padding: 1vw;
-}
-
-.ul {
-  display: flex;
-  gap: 1vw;
-  list-style: none;
-}
-
-.li-app {
-  text-decoration: none;
-  color: black;
-}
-
-h1 {
-  color: rgb(64, 91, 213);
-}
-
-.ul-books {
-  display: flex;
-  flex-direction: column;
-  justify-content: space-between;
-  padding: 2vw;
-  list-style: none;
-}
-
-.li-books {
-  display: flex;
-  justify-content: space-between;
-}
+.Nav-bar {
+  padding: 24px 4%;
+  background-color: #fff;
+  border-bottom: 1px solid #e8e8e8;
+  display: flex;

```

Comment:
- [ ] You have used  **display: flex**  10 times which means you are repeating yourself a lot and that is bad CSS practice. I would suggest you create a class like d-flex and use this at any HTML component that you need display flex
![image](https://user-images.githubusercontent.com/62705947/215122351-ba9a91cd-b9d6-4cea-b939-a241b4763c15.png)